cogs/welcome/welcome.py

```diff
@@ -97,14 +99,45 @@ async def sendWelcomeMessageChannel(self, newUser: discord.Member, guild: discor
             message += splitMessage[x]
             if x < len(splitMessage) - 1:
                 message += newUser.mention
-        fullMessage = f"""{message} Welcome to SFU Anime Club's Discord!
 
-Please check out <#225044755887685632> for all of our basic information and rules, and feel free to introduce yourself with the pinned template in this channel!
-
-We hope you enjoy your stay~
+        joinLogEnabled = await self.config.guild(newUser.guild).logJoinEnabled()
+        joinLogChannel = await self.config.guild(newUser.guild).logJoinChannel()
+        errorChannel = discord.utils.get(
+            newUser.guild.text_channels, id=joinLogChannel
+        )
+        try:
+            await channel.send(message)
+        except (discord.Forbidden, discord.HTTPException) as errorMsg:
+            LOGGER.error(
+                "Could not send message, the user may have"
+                "turned off DM's from this server."
+                " Also, make sure the server has a title "
+                "and message set!",
```

Comment:
I think this is sufficient:
```suggestion
                "Could not send message, please make sure the bot "
                "has enough permissions to send messages to this "
                "channel!"
```

Reply:
Done